### ISSUE TITLE
Support xyz format?

### ISSUE THREAD

**Issue by hainm** [arose/ngl#649]
Hi, currently NGL does not support .xyz format yet (It's kind of free format). This format is popular (?) in several packages (e.g: psi4, qcelemental, ase). 
https://en.wikipedia.org/wiki/XYZ_file_format

Example:
```
3
water
 H    0.7838370000   -0.4922360000   -0.0000000000
 O   -0.0000000000    0.0620200000   -0.0000000000
 H   -0.7838370000   -0.4922360000   -0.0000000000
```

**Comment by fredludlow** [comment]
Good idea, I won't have time in the near future but it would make a good first PR if anyone else out there wants to get involved!
